druid-shell/src/platform/x11/application.rs

```diff
@@ -87,14 +91,81 @@ impl Application {
             quitting: false,
             windows: HashMap::new(),
         }));
+
+        let present_opcode = if std::env::var_os("DRUID_SHELL_DISABLE_X11_PRESENT").is_some() {
```

Comment:
We should document such toggles somewhere, maybe just in an `# Env` section at the crate root of `druid-shell`.